modules/ob-examples-of-tekton-resources-management.adoc

```diff
@@ -0,0 +1,146 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
```

Comment:
Would you say that this module is a procedure or a concept? I'm leaning more towards procedure to obtain info about resources. This section probably needs revised to be revised to be more like a procedure on how to obtain information about Tekton resources but running/entering the attached `oc` commands. Each procedure would need to be its own module, so the content would need moved. 

I'm conflicted, though, on what it is and how best to approach it. My .2c is to approach it as a procedure.

Reply:
I would call this a conceptual stuff because a user does not need to perform any procedure in context of Tekton resource management. 
This conceptual module basically focuses on how Tekton performs resource allocation for a build step based on the values a user defined in a build startegy CR (the same is explained in different scenario-based examples). The configuration examples for defining build step resources has already been discussed in the `defining step resources` section. 
This partcular section `Resource management in tekton Pipelines` actually highlights the role of Tekton in resource allocation based on the values a user defined in a build strategy. After going through this module, a user can build an understanding around defining recource limit for a particular build step.